agenta-cli/agenta/cli/main.py

```diff
@@ -101,8 +102,14 @@ def init(app_name: str):
                 "Please provide the IP or URL of your remote host"
             ).ask()
         elif where_question == "On agenta cloud":
-            backend_host = "https://cloud.agenta.ai"
-            api_key = helper.get_api_key()
+            backend_host = questionary.text(
+                "Please provide the URL for the backend host. Defaults to: https://cloud.agenta.ai",
```

Comment:
let's remove this and use the config version per default.

Reply:
Okay, okay.